### PR TITLE
Merging template updates 2.0.1

### PR DESCRIPTION
**Pull request by skrakau**
Merge template updates 2.0.1:

- changed a few output paths due to `publish_by_id` -> `publish_by_meta` and adjusted this accordingly in the `output.md` (the generation of custom paths specified before with`publish_id`: 'bla_${meta.id}' did not work anymore like this)
- adjusted `.editorconfig` as in template `dev` branch to avoid lint test to fail (https://github.com/nf-core/tools/commit/d7ebfa11b7b4c54c131f393cb22f5188f44e4eba)
- and changed path to testdata to `https://raw.githubusercontent.com/nf-core/test-datasets` instead of using `https://github.com/nf-core/test-datasets/raw`

(Note, I accidentally started the template updates on the wrong branch containing commits regarding Prodigal which are now already merged into `dev` https://github.com/nf-core/mag/pull/217.  However, GitHub doesn't detect those changes in `dev`, but it should be fine when merging. So the commits `Publish genes predicted with Prodigal in BUSCO process`, `Mention prodigal output in docs`, `Update CHANGELOG` can be ignored.)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/mag/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -1,121 +1,145 @@
...
+    singularity_pull_docker_container = false
+
+    // References
+    igenomes_base              = 's3://ngi-igenomes/igenomes'
+    igenomes_ignore            = false
+
+    // MultiQC options
+    multiqc_config             = null
+    multiqc_title              = null
+    max_multiqc_email_size     = '25.MB'
```
Comment (ggabernet) [inline nextflow.config:112]:
```suggestion
```

Comment (ggabernet) [reply nextflow.config:112]: They are duplicated

File: nextflow.config [hunk 2]
```diff
@@ -1,121 +1,145 @@
...
+    email                      = null
+    email_on_fail              = null
+    plaintext_email            = false
+    monochrome_logs            = false
+    help                       = false
+    validate_params            = true
+    show_hidden_params         = false
+    schema_ignore_params       = 'genomes,modules'
+    enable_conda               = false
+    singularity_pull_docker_container = false
```
Comment (ggabernet) [inline nextflow.config:127]:
```suggestion
```

Comment (ggabernet) [reply nextflow.config:127]: Also duplicates

File: nextflow.config [hunk 3]
```diff
@@ -1,121 +1,145 @@
...
+
+    // References
+    igenomes_base              = 's3://ngi-igenomes/igenomes'
+    igenomes_ignore            = false
```
Comment (ggabernet) [inline nextflow.config:107]:
```suggestion
```

Comment (ggabernet) [reply nextflow.config:107]: They are duplicated

**Review by ggabernet — APPROVED** [review]
Hi @skrakau, it is looking good to me, I just had a couple of comments :)

**Comment by skrakau** [comment]
Thanks a lot @ggabernet !

**Pull request merged**